### ISSUE TITLE
[Package Issue]: Governikus.AusweisApp

### ISSUE THREAD

**Issue by CanePlayz** [microsoft/winget-pkgs#139155]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Other

### Brief description of your issue

winget does not properly detect my installation of AusweisApp, despite the manifest having the correct product code associated with it.

### Steps to reproduce

Basically just install the app

### Actual behavior

This is the output from `winget list AusweisApp`:

```
Name       ID                                     Version
---------------------------------------------------------
AusweisApp {D2AD1FE6-DD7E-441C-BECE-A492D126E1E2} 2.0.3
```

Note that the winget ID does not appear and that winget isn't shown as the source (that's what happens with properly picked up packages).

### Expected behavior

When I do `winget list`, I would expect winget to show the winget id and winget as the source.

### Environment

```raw
Windows Package Manager v1.6.10121
Windows: Windows.Desktop v10.0.26052.1000
```

**Comment by CanePlayz** [comment]
Other packages that cannot be matched currently are Spotify.Spotify and WhatsApp.Beta, but before I open issues for them for nothing, are these known issues because they are apps installed from the MS store with very regular updates?

**Comment by mdanish-kh** [comment]
That's odd, I couldn't reproduce it in my VM. Is there a chance you can uninstall the package and re-install via winget? Can you also share the logs from `winget list AusweisApp --verbose --open-logs` (should be the most recent log)

**Comment by CanePlayz** [comment]
Yeah, I guess that's happening with all packages installed from msstore as I just noticed that it's also the case for Asus.ArmouryCrate. These apps don't show their version in "Installed apps" and don't show up in my ARPTable so I guess it's just down to them being registered differently in Windows which prevents winget from picking them up properly?
